runtime/doc/develop.txt

```diff
@@ -233,9 +233,9 @@ definitions.  The |lua-vim| :help is generated from the docstrings.
 
 Docstring format:
 - Use LuaLS annotations: https://github.com/LuaLS/lua-language-server/wiki/Annotations
```

Comment:
This _needs_ to be qualified now (as well as replaced by https://luals.github.io/wiki/annotations/).

Reply:
I don't understand what qualified means in this context lol. Mind using the suggestion feature to write what you'd like?